pages/committees/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Head from "next/head";
+import { Wrapper } from "../../styles/containers";
+import HeaderNav from "../../components/headerNav";
+import Jumbotron from "../../components/jumbotron";
+import { Body } from "../../styles/typography";
+import Footer from "../../components/footer";
+import Announce from "../../components/Announce";
+import { InformationalWrapper } from "../../app/components/elements";
+import CommitteeDisplay from "../../app/components/CommitteeDisplay";
+
+const Committees = () => {
+  return (
+    <Wrapper>
+      <Head>
+        <title>Committees & Cabinets - CIMUN XVIII</title>
+        <meta
+          name="description"
+          content="An Overview of the Committees & Cabinets at CIMUN XIX - February 2nd-5th, 2023 at the Palmer House Hilton Chicago"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Mulish:wght@900&display=swap"
+          rel="stylesheet"
+        />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Alegreya:wght@600&display=swap"
+          rel="stylesheet"
+        />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@500&family=Alegreya:wght@600&display=swap"
+          rel="stylesheet"
+        />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@500&family=Alegreya:wght@600&family=Source+Sans+Pro&display=swap"
+          rel="stylesheet"
+        />
+      </Head>
+      <Announce.RegistrationOpen />
+      <HeaderNav />
+      <Jumbotron
+        titleOne="COMMITTEES"
+        titleTwo="& CABINETS"
+        subTitle="CIMUN XIX"
+        height={85}
+        image="awards.jpg"
+        bottomMargin="0.984375vw"
+      />
```

Comment:
didn't I change the jumbotron component? did that never get merged in?

Reply:
not yet lol